### PR TITLE
Update to icp4d version 3.0

### PR DESCRIPTION
**Pull request by ljbennett62**
updated up to Step 4 - to ICP4D 3.0 and also to match workshop. Need to discuss step 4 w/u Scott - seems to diverge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,65 +83,77 @@ Cloud Pak for Data uses the concept of `Deployment Spaces` to configure and mana
...
+![Deployment space collaborators](doc/source/images/deploymentSpaceCollaborators.png)
 
 ## 3. Configure OpenScale in a Jupyter Notebook
 
 For this part of the pattern we're going to configure our Watson OpenScale service by running a Jupyter Notebook.
 
 ### Import the notebook
 
-At the project overview, either click the *New Asset* button [1], and choose *Add notebook* or go to the *Notebooks* tab [2] and choose *+ Add notebook* [3].
+At the project overview click the Add to project button, and choose Notebook or click New notebook option next to the Notebooks section.
```
Comment (markstur) [inline README.md]: This line no longer adds emphasis to the buttons names (or other screen elements to look for) with backticks (or italics or bold -- our style varies even in this one doc).  I.e. `Add to project`, `Notebook`, `New`, `Notebooks` and also the following line has `From URL`.

**Review by markstur — COMMENTED** [review]
nit: some emphasis for screen elements got lost in 2 lines

**Review by markstur — APPROVED** [review]
LGTM. I added some back-ticks to address my own comment.  @scottdangelo review?

**Review by scottdangelo — APPROVED** [review]
LGTM

**Pull request merged**